File: Training/URI/categoria-Iniciante/C/1180.c
#include<stdio.h>
#include<stdlib.h>

main() {
	int n;
	int *vetor;
	int i;
	int menor;
	int posicao;
	scanf("%d",&n);

	vetor = (int*) malloc(n * sizeof(int));


	for(i = 0; i < n; i++) {
		scanf("%d",&vetor[i]);
	}
	menor = vetor[0];
	posicao = 0;
	for(i = 1; i < n; i++) {
		if(vetor[i] < menor) {
			menor = vetor[i];
			posicao = i;
		}
	}

	printf("Menor valor: %d\n", menor);
	printf("Posicao: %d\n", posicao);


	free(vetor);
}

File: Training/URI/categoria-Iniciante/java/1180.java

import java.util.Scanner;;

public class Main {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		int n = scanner.nextInt();
		int menor, posicao;
		int[] vetor = new int[n];

		for(int i = 0; i < n; i++) {
			vetor[i] = scanner.nextInt();
		}

		menor = vetor[0];
		posicao = 0;

		for(int i = 1; i < n; i++) {
			if(vetor[i] < menor) {
				menor = vetor[i];
				posicao = i;
			}
		}

		System.out.println("Menor valor: " + menor);
		System.out.println("Posicao: " + posicao);

	}
}
